OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -47,12 +47,12 @@ public class AircraftInfo : PausableConditionalTraitInfo, IPositionableInfo, IFa
 		[Desc("The speed at which the aircraft is repulsed from other aircraft. Specify -1 for normal movement speed.")]
 		public readonly int RepulsionSpeed = -1;
 
-		public readonly int InitialFacing = 0;
+		public readonly WAngle InitialFacing = WAngle.Zero;
 
-		public readonly int TurnSpeed = 255;
+		public readonly WAngle TurnSpeed = new WAngle(1023);
 
 		[Desc("Turn speed to apply when aircraft flies in circles while idle. Defaults to TurnSpeed if negative.")]
```

Comment:
"Defaults to TurnSpeed if undefined".

This is a trivial change, so will leave it until there are other review comments.